contrib/ci/fedora-openssl.sh

```
@@ -0,0 +1,27 @@
+#!/bin/sh
+set -e
+set -x
+
+#check for and install missing dependencies
+./contrib/ci/fwupd_setup_helpers.py install-dependencies --yes -o fedora
+
+# make sure gnutls is not available
+sudo dnf remove -y gnutls-devel
+
+root=$(pwd)
+export BUILD=${root}/build
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)